plugins/backend/backstage-plugin-aws-auth/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@roadiehq/backstage-plugin-aws-auth",
+  "version": "0.3.4",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "start": "backstage-cli backend:dev",
+    "build": "backstage-cli backend:build",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test --passWithNoTests",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "backstage": {
+    "role": "backend-plugin"
+  },
+  "repository": {
+    "type": "git",
+    "url": "github:RoadieHQ/roadie-backstage-plugins",
+    "directory": "plugins/backend/backstage-plugin-argo-cd-backend"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.9.7",
+    "@types/express": "^4.17.11",
+    "aws-sdk": "^2.902.0",
```

Comment:
We'll probably want to migrate to a later version (eg v3). I'd say it should be another task though